javascript/lint/action.yaml

```diff
@@ -0,0 +1,43 @@
+---
+name: "Lint JavaScript, JSON, YAML and Markdown"
+description: "Runs a couple of linters"
+
+inputs:
+  install-command:
+    required: false
+    description: "Command used to install linters"
+    default: "npm install prettier eslint"
```

Comment:
Imo you should pin prettier and eslint to some version by default as it can still be overrodeed via install-command . As recently we have problem with unpined "black" linter that force us to reformat half of repository as it do braking change.

Reply:
Good idea, I have done that just now